packages/trader/src/Modules/Trading/Components/Elements/__tests__/purchase-button.spec.tsx

```diff
@@ -75,20 +89,32 @@ describe('<PurchaseButton />', () => {
     });
 
     it('should render the button with <Money /> component inside for multipliers contract', () => {
-        render(<PurchaseButton {...default_mocked_props} is_multiplier type='MULTUP' />);
+        render(
+            <StoreProvider store={mock_store}>
+                <PurchaseButton {...default_mocked_props} is_multiplier type='MULTUP' />
+            </StoreProvider>
+        );
 
         expect(screen.getByText(/UP/i)).toBeInTheDocument();
```

Comment:
just a nitpick:
I see, that everywhere in the PR we are taking elements by text, and text is a regexp with 'i'. we can not use capital letters with this 'i'
but you can ignore this :)